### ISSUE TITLE
Phillip's sieges

### ISSUE THREAD

**Issue by PitchNeeded** [Sh0wdown/UnofficialCrusaderPatch#272]
I think Phillip's spearmen should be made to just advance immediately once they gather rather than advancing a little bit then waiting, as they do now. Because Phillip's sieges only consist of spearmen, and no siege equipment is created or missile troops advancing first, the waiting stage for the spearmen is pointless and leads to them being vulnerable to missile fire.

As for other AIs siege wait times, most of them are OK I think, apart from Pig's macemen, who wait a bit too long before advancing I think during sieges (probably because he waits for the crossbowmen who are slow), and their wait time should be shortened I think, as standing macemen are vulnerable to arrow fire.

Also of course there was a debate on here before as to whether Phillip should have 'proper' sieges. He was King of France and so his sieges do seem a bit pathetic. If he used knights, spearmen, archers and catapults that would make his sieges unique enough from Rat and Snake and a bit more threatening (and he does sometimes say 'my knights will ride you down' before sieges). Using knights in sieges would also fit with his in game description, which mentions 'his reckless insistence on the use of his knights as his main fighting force'. 

However on the other hand maybe just sending spearmen and not putting much effort into sieges fits with his lazy personality: 'Oh, I lose interest in this silly conflict'.

**Comment by Truetobi91** [comment]
> 
> the waiting stage for the spearmen is pointless and leads to them being vulnerable to missile fire.

To be honest, I'm pretty sure that won't change a thing. In testing (my modded) Philipp advanced their spearmen immediately often enough but that did'nt really change the fact that they wouldn't arrive at the enemy's castle anyway due to their low arrow resistance (even with buff) and their average movement speed. Testing against was done against: Philipp (mirror match), Rat, Sultan, Snake, Pig, Caliph

This is not an exclusive Philipp problem but a general spearmen problem but due to his low numbers, Phillip doesn't benefit much from the increased army limit. I'd suggest to either further buff spearmen for a higher recruiting price of 12 to 15 gold depending on how many changes are done:
- Increase arrow resistance from 5 to 7
- Have running speed as consistent as Macemen
- Increase damage against civilists, cows and oxen 

OR

- Add other units (knights/archers/catapults) to his attacks (as you suggested),

OR 

- Force Philipp to create pikemen instead of spearmen and slightly rebalance his AIVs.


> 
> However on the other hand maybe just sending spearmen and not putting much effort into sieges fits with his lazy personality: 'Oh, I lose interest in this silly conflict'.

That's absolutely legitimate imo but I found his current offensive power to be problematic in 1v1 scenarios. Despite losing 0:2 against both snake and caliph in my latest tests he did pretty good in several other games against them, thriving them against the wall but being absolutely unable to even touch their castles because
- the few catapults he builds tend to get stuck too easily despite being build outside of the castle 
- his knights don't attack the castle on purpose
- his spearmen don't stand a chance against a snake or caliph at 50% garrison

resulting in matches drawing out for hours in 1000 speed without any progress. He should have definately be able to take a victory here or there.

**Comment by Panbutt** [comment]
I wholly agree that being the King of France and part of the "crusader trifecta" along with Frederick and Lionheart, Philly needs some beefing up.

Pikemen are actually an interesting idea tbh. But I'm thinking he should use -both- Spearmen and Pikemen. Pikemen are tanky and in high numbers, spearmen can do crazy amounts of damage. Phillip also naturally builds poleturners and armorers so this does not require any AIV edits.

That said, he shouldn't use too many pikemen. He is an 'easy' Lord afterall. There should be a 3:1 recruitment ratio for spearmen and pikemen. I'm also on board with him using archers and catapults in his attacks.

**Comment by Monsterfisch** [comment]
pikemen don't strike me as a french kind of unit in the historical sense (i believe I touched upon that topic in a different issue before)  i would argue crossbowmen would be more in line for Phillip considering it would mean cheese farms and we all know french like cheese. On top of that crossbows have been used by the French during that time I believe while the English had their longbows etc :) it would also make for a very interesting unit composition hehe.
that being said considering from what I heard a tool/config is in the works so the player can modify aiv behavior themselves which will open up the floodgates and make for some exciting scenarios.

**Comment by PitchNeeded** [comment]
> that being said considering from what I heard a tool/config is in the works so the player can modify aiv behavior themselves which will open up the floodgates and make for some exciting scenarios.

Oh this sounds great! :D

**Comment by Zer0oZ** [comment]
> 
> 
> pikemen don't strike me as a french kind of unit in the historical sense (i believe I touched upon that topic in a different issue before) i would argue crossbowmen would be more in line for Phillip considering it would mean cheese farms and we all know french like cheese. On top of that crossbows have been used by the French during that time I believe while the English had their longbows etc :) it would also make for a very interesting unit composition hehe.

I agree. I'd rather see Phillip use swordsmen (essentially dismounted knights so they'd fit), spearmen (weak levy troops) and crossbows ("french" crossbows against english longbows was a thing in the famous battle of Crecy, would be a nice anecdote).  Phillip would probably be alot more powerful at this point with these units+knights but its nice to think about all the new stuff we can experiment with once the new customization options are available :)

I'd love to swap Marshals swordsmen with pikemen, its a unit thats underrepresented imo.

**Comment by PitchNeeded** [comment]
> that being said considering from what I heard a tool/config is in the works so the player can modify aiv behavior themselves which will open up the floodgates and make for some exciting scenarios.

Oh this sounds great! :D Personally, I wouldn't want to change too many AIs tactics though, it would take away from their own unique personality. But small things like defensive slingers for Saladin and better sieges for Phillip would be good to impliment.
